CMakeLists.txt

```diff
@@ -32,6 +32,10 @@ if(NOT WIN32 AND NOT APPLE AND NOT HAIKU)
   option(ENABLE_EGL "Enables EGL OpenGL Interface" ON)
 endif()
 
+if(NOT ANDROID)
+  option(ENABLE_CLI_TOOL "Enable dolphin-tool, a CLI tool for power users" ON)
```

Comment:
Specifying "for power users" doesn't add anything useful in my opinion.

Maybe "Enable dolphin-tool, a CLI tool for managing disc images"?